### DISCUSSION TITLE
How to add (any/current) song to (arbitrary) playlist ?

### DISCUSSION THREAD

**Discussion by shula** [audacious-media-player/audacious#1520] [General]
How to copy/add/move a song (from current playlist)  to another playlist, via  GUI  ?

Ideally, such a tool would let choose a target playlust  (either via context sub-menus or a small popup/dialog.)

(Yes, I know I can do that via `audtool`.)

**Comment by radioactiveman** [comment]
Simply copy or cut a song from one playlist, switch to the other playlist and paste it there.

**Reply by shula** [reply]
Thank you.

BTW, This is not presented in the GUI in any way; maybe it should?

**Reply by mschwendt** [reply]
> BTW, This is not presented in the GUI in any way;

What makes you think so?

Using the GUI you can switch between playlists, create new playlists and copy or cut'n'paste tracks between playlists.

**Reply by shula** [reply]
I'm saying it's not obvious by looking at the main menus, or the context menu, or via configurable shortcuts.
It's not even available directly via audtool     @mschwendt  
Which is why I came up with this question in the 1st place.

**Reply by krukai** [reply]
IMO, copy & paste operations aren't necessarily the assumed go-to operations in this context. I stumbled over this discussion because I was surprised that a simple drag'n'drop does not work. I would have expected that dragging a song (or multiple) from the currently active playlist onto another playlist would perform the copy operation, and perhaps a move (i.e., cut) if some modifier is held. Would that be possible?

**Reply by radioactiveman** [reply]
I assume you use the Qt interface, right?

With the GTK interface it is possible to drag and drop songs to another playlist.
Moving the cursor on the playlist tab changes to this playlist where you then can drop the songs.

The Qt interface does not respond to this event. Feel free to create a feature request in our bug tracker.
